src/chunkserver/copyset_node.cpp

```diff
@@ -526,7 +526,7 @@ int CopysetNode::LoadConfEpoch(const std::string &filePath) {
                        << ", Copyset: " << GroupIdString();
             ret = -1;
         } else {
-            epoch_.store(loadEpoch, std::memory_order_relaxed);
+            epoch_.store(loadEpoch, std::memory_order_release);
```

Comment:
LoadConfEpoch is called on copyset in initialization，there is no competition at this time.